tests/e2e/src/create_test_data/createTestDataManage.js

```
@@ -0,0 +1,90 @@
+  const dataStr = JSON.stringify(data, null, 4);
+  if (dataStr) {
+    try {
+      fs.writeFileSync(filePath, dataStr);
+    } catch (err) {
+      // eslint-disable-next-line no-console
+      console.log(`save error ${err}`);
+    }
+  }
+};
+const accountFrom = web3.eth.accounts.privateKeyToAccount(Config.getIns().hexPrivateKey);
+const sendTransaction = async (account, data) => {
+  const nonce = (await web3.eth.getTransactionCount(accountFrom.address)) + 1;
+  const tx = {
+    type: 2,
+    nonce,
+    maxPriorityFeePerGas: 2500,
+    maxFeePerGas: 2500,
+    gasLimit: web3.utils.stringToHex("21000"),
+    chainId: Config.getIns().axonRpc.chainId,
+    data,
+  };
+  const transaction = await accountFrom.signTransaction(tx);
+  return web3.eth.sendSignedTransaction(transaction.rawTransaction);
+};
+
+const createTransactionData = async () => {
+  const contract = new web3.eth.Contract(erc20.abi);
+  const txOptions = { data: erc20.bytecode, arguments: ["TT", "TTT"] };
+  const data = contract.deploy(txOptions).encodeABI();
+  const receipt = await sendTransaction(
+    accountFrom.address,
+    data,
+  );
+  transactionInfo.contractAddress = receipt.contractAddress;
+  transactionInfo.transactionHash = receipt.transactionHash;
+  transactionInfo.blockHash = receipt.blockHash;
+  transactionInfo.blockNumber = receipt.blockNumber;
+  transactionInfo.transactionIndex = receipt.transactionIndex;
+  transactionInfo.accountAddress = accountFrom.address;
+  transactionInfo.hexBlockNumber = `0x${receipt.blockNumber.toString(16)}`;
+  await savejson(`${basePath}/testData_1.json`, transactionInfo);
+};
+
+const readTestDataAsJson = (testFileName) => {
+  let data;
+  try {
+    const filePath = `${basePath}/${testFileName}`;
+    const jsonData = fs.readFileSync(filePath, "binary").toString();
+    data = JSON.parse(jsonData);
+  } catch (err) {
+    // eslint-disable-next-line no-console
+    console.log(err);
+  }
+  return data;
```

It might no be a good idea to hide the exception.